candidate-standard/abstract_tests/ATS_class_collections.adoc

```diff
@@ -23,38 +23,30 @@ include::collections/ATS_rc-md-success.adoc[]
 [[collections-metadata-schema]]
 include::collections/ATS_rc-md-success-table.adoc[]
 
-==== Feature Collection {root}/collections/{collectionId}
+==== Environmental Data Collection {root}/collections/{collectionId}
 
 include::collections/ATS_src-md-op.adoc[]
 
 [[ats_collections_src-md-success]]
 include::collections/ATS_src-md-success.adoc[]
 
-==== Features {root}/collections/{collectionId}/items
-
-NOTE: This test is too Feature centric. Will need to be greatly reduced in scope.
-
-include::collections/ATS_rc-op.adoc[]
-
-include::collections/ATS_rc-response.adoc[]
-
-==== Second Tier Tests
+==== Second Tier Collections Tests
 These tests are invoked by other tests.
 
-===== Extent
+===== Collection Extent
```

Comment:
We may want to add more context to headings like this to make it clear what we are talking about in each section.

Reply:
agreed. Context is All.